tests/core/data/test_parquet_data_node.py

```diff
@@ -589,9 +590,7 @@ def test_partition_cols(self, tmpdir_factory, default_data_frame: pd.DataFrame):
         temp_dir_path = str(tmpdir_factory.mktemp("data").join("temp_dir"))
 
         write_kwargs = {"partition_cols": ["a", "b"]}
-        dn = ParquetDataNode(
-            "foo", Scope.SCENARIO, properties={"path": temp_dir_path, "write_kwargs": write_kwargs}
-        )  # type: ignore
+        dn = ParquetDataNode("foo", Scope.SCENARIO, properties={"path": temp_dir_path, "write_kwargs": write_kwargs})  # type: ignore
```

Comment:
Line is too long!

Reply:
It's fit the current 120 maximum length rule though

Reply:
134 characters, including the comment.

Reply:
oh it's probably because of the `# type: ignore`. I'm removing a lot of those and see which can be fixed

Reply:
Oh it is because it doesn't count the `# type: ignore` comment

Reply:
Let's count the comments as well. No?

Reply:
I will try to fix the mypy first to avoid the `# type: ignore`, then find a way latter 😄 I do not know how to count comments yet but will do

Reply:
TODO added